charts/opentelemetry-demo/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.26.0
+version: 0.27.0
```

Comment:
This is not a functional PR, I don't think we need to update the minor version, how about `0.26.1`?